### PR TITLE
Added comments controller

### PR DESCRIPTION
**Pull request by diogobroco**
First controller for comments (it could be changed and updated in the future, depending on the docs and the commontator gem)

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    @comment = Comment.new(comments_params)
+    @doc = Doc.find(params[:id])
+    @user = current_user
+    @comment.doc = @doc
+    @comment.user = @user
+
+    if comment.save
+      redirect_to @doc
+    else
+      render :new # should be 'docs/show'
```
Comment (Tiago-O) [inline app/controllers/comments_controller.rb]: acho que podes escrever logo:
render 'doc/show'

**Review by Tiago-O — COMMENTED** [review]
falta fazer o edit, update e destroy

**Comment by diogobroco** [comment]
Check the new updates

**Review by Tiago-O — APPROVED** [review]
looks good!

**Pull request merged**